test/test-network/systemd-networkd-tests.py

```diff
@@ -2710,6 +2711,49 @@ def test_bind_carrier(self):
         self.assertRegex(output, 'inet 192.168.10.30/24 brd 192.168.10.255 scope global test1')
         self.wait_operstate('test1', 'routable')
 
+    def test_activation_policy(self):
+        for test in ['up', 'always-up', 'manual', 'always-down', 'down', '']:
```

Comment:
How about to use `self.subTest()`?

Reply:
it does use subTest() in the up/down/up/down iterations further down in the test, but let me check if it's possible to nest subTests

Reply:
the nested subTest() calls seem to work great, thanks for the suggestion!